app/presenters/publishing_api_presenters/organisation.rb

```diff
@@ -16,7 +16,9 @@ def details
 private
 
   def crest
-    item.organisation_logo_type.class_name
+    class_name = item.organisation_logo_type.class_name
+    is_publishable_crest = class_name != "no-identity" && class_name != "custom"
```

Comment:
I think this might be clearer if you extract the 'is_publishable...' bit out into
```
def crest_is_publishable?
  item.class_name != etc
end
```
and then the `crest` method becomes one line...

`crest_is_publishable? ? etc...`